synapseclient/__main__.py

```diff
@@ -232,16 +232,6 @@ def associate(args, syn):
             print('%s.%i\t%s' % (ent.id, ent.versionNumber, fp))
 
 
-def copy(args, syn):
```

Comment:
Please add a Todo back to the Jira ticket to document the removal of `copy` function from the command line client in the Synapse Python client v2.0 release notes. Please also mark the Synapse Python client v2.0 epic as blocked and link to this ticket as "blocked by SYNPY-999."